### PR TITLE
zh-trans: add docs/tasks/configure-pod-container/share-process-namespace.md

### PR DESCRIPTION
**Pull request by SataQiu**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/share-process-namespace.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+title: 在 Pod 中的容器之间共享进程名称空间
```
Comment (xichengliudui) [inline content/zh/docs/tasks/configure-pod-container/share-process-namespace.md]: 命名空间

Comment (chenrui333) [reply content/zh/docs/tasks/configure-pod-container/share-process-namespace.md]: 👍

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @SataQiu, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/661) had been assigned to others. will not be reported to the task issues

File: content/zh/docs/tasks/configure-pod-container/share-process-namespace.md (hunk 1 above)
Comment (SataQiu) [reply content/zh/docs/tasks/configure-pod-container/share-process-namespace.md]: Done!

**Pull request merged**